physionet-django/events/models.py

```diff
@@ -37,27 +62,107 @@ def save(self, *args, **kwargs):
     def __str__(self):
         return self.title
 
-    def enroll_user(self, user):
+    def waitlist(self, user, comment=None):
```

Comment:
Here, the idea for `waitlist` was  more like "waitlist this user to the event, since the method is in the Event`

Wouldn't `join_waitlist` be more appropriate if the method was in `EventParticipant`?